src/Makefile

```diff
@@ -21,6 +21,7 @@ FLAGS := \
 	-D_GNU_SOURCE -I$(BUILDDIR) -I$(SRCDIR) \
 	-I$(SRCDIR)/flisp -I$(SRCDIR)/support \
 	-I$(LIBUV_INC) -I$(build_includedir) -DLIBRARY_EXPORTS \
+	-I$(LOCALBASE)/include \
```

Comment:
ah yeah see the appveyor failure, unconditionally adding this is bad for cases when you're cross-compiling